File: server/src/main/java/dev/dtos/responses/ProductResponse.java
package dev.dtos.responses;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class ProductResponse {
    private String  id;
    private String name;
    private String  category;
    private String  imageURL;
    private String  description;
    private Integer quantity;
    private Double  price;
}

File: mobile/app/src/main/java/com/javaproject/mobile/data/model/responses/ProductResponse.kt
package com.javaproject.mobile.data.model.responses

import com.google.gson.annotations.SerializedName

data class ProductResponse (
    @SerializedName("id")
    val id: String,
    @SerializedName("name")
    val name: String,
    @SerializedName("category")
    val category:String,
    @SerializedName("imageURL")
    val imageURL: String,
    @SerializedName("quantity")
    val quantity: Int,
    @SerializedName("price")
    val price: Double,
    @SerializedName("description")
    val description: String
)